src/hooks/helper/index.ts

```diff
@@ -0,0 +1,4 @@
+import { SubmittableExtrinsic, SubmittableExtrinsicFunction } from '@polkadot/api/types';
+import { ISubmittableResult } from '@polkadot/types/types';
+
+export type BatchTxs = SubmittableExtrinsic<'promise', ISubmittableResult>[];
```

Comment:
I see that you try to make the type `BatchTxs` as an array of `SubmittableExtrinsic`, but please note that a batch transaction when it's submitted to the chain is considered as a single `SubmittableExtrinsic` with the input of `SubmittableExtrinsic[]`, this might be confusing for others (see https://github.com/hoonsubin/substrate-batch-script/blob/main/src/api/SubstrateApi.ts#L113)

Reply:
[Fixed](https://github.com/AstarNetwork/astar-apps/pull/258/commits/1e7f40e568c4c5b1162767efcb80cf8cb0436109)